src/python/doc/bottleneck_distance_user.rst

```diff
@@ -35,19 +35,19 @@ The following example explains how the distance is computed:
 
     import gudhi
 
-    message = "Bottleneck distance = " + '%.1f' % gudhi.bottleneck_distance([[0., 0.]], [[0., 13.]])
+    message = "Bottleneck distance = " + '%.1f' % gudhi.bottleneck_distance([[0., 2.]], [[1., 13.]])
```

Comment:
If you change the example, the [figure](https://gudhi.inria.fr/python/latest/bottleneck_distance_user.html#distance-computation) is no more correct

Reply:
Indeed. I've reverted that.